.gitignore

```diff
@@ -14,6 +14,7 @@ stdout*
 .idea/
 docs/_build/
 coverage.xml
+venv/
```

Comment:
I'll remove it, but I consider using the global .gitignore for anything is a bad idea. You never know when it will conflict with the needs of some project, at which point you need a custom `!foo` line in the project .gitignore or `!/path/to/project/foo` in the global .gitignore.